src/Orckestra.Composer.Website/App_Data/Razor/Composer/SingleCheckout/Confirmation.cshtml

```diff
@@ -92,4 +104,55 @@
             </div>
         </div>
     </div>
+}
+
+@helper CreateAccountForm(PasswordPatternViewModel passwordPatternViewModel)
+{
+    <form v-if="ShowCreateAccountForm" class="mt-4" id="formCreateAccount">
+        <h3>@Html.Localize("CheckoutProcess", "T_SaveInformation")</h3>
+        <p>@Html.Localize("CheckoutProcess", "L_CreateAccountForFastCheckout")</p>
+        <div class="form-row align-items-end">
+            <div class="form-group col-md-6">
+                <label for="Password">@Html.Localize("Forms", "F_Password") *</label>
+                <input v-model="Password"
+                       id="Password"
+                       name="Password"
+                       data-qa="information-password-input"
+                       v-bind:disabled="IsLoading"
+                       v-on:input="onChangePassword"
+                       v-bind:type="ShowPassword ? 'text' : 'password'"
+                       class="form-control"
+                       required="required"
+                       data-parsley-length="[@passwordPatternViewModel.MinRequiredPasswordLength, 256]"
+                       data-parsley-pattern="@passwordPatternViewModel.PasswordRegexPattern"
+                       @Html.ParsleyMessage("Forms", "F_PasswordRequiredError", "required-message")
+                       @Html.ParsleyMessage("Forms", "F_PasswordLengthError", "length-message")
+                       @Html.ParsleyMessage("Forms", "F_PasswordPatternError", "pattern-message", passwordPatternViewModel.MinRequiredNonAlphanumericCharacters)
+                       @Html.HelpBubble("Forms", "F_PasswordIndication", passwordPatternViewModel.MinRequiredPasswordLength) />
+            </div>
+            <div  class="form-group p-2 col-auto">
+                <a href="#" role="button" v-on:click="showPasswordToggle" class="text-decoration-none">
```

Comment:
add prevent as now it jumps to the top

v-on:click.prevent="showPasswordToggle"